google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/RowMutation.java

```diff
@@ -34,12 +35,18 @@
 public final class RowMutation implements MutationApi<RowMutation>, Serializable {
   private static final long serialVersionUID = 6529002234913236318L;
 
-  private final String tableId;
+  private final TargetId targetId;
   private final ByteString key;
   private final Mutation mutation;
 
-  private RowMutation(String tableId, ByteString key, Mutation mutation) {
-    this.tableId = tableId;
+  private RowMutation(@Nonnull String tableId, ByteString key, Mutation mutation) {
```

Comment:
Same for this class, remove this ctor, no need for Nonnull, and add @Deprecated annotation with some comments.